### PR TITLE
SURF-812: Refactor/Reorganize Docs

### PR DESCRIPTION
**Pull request by CITguy**
JIRA: https://jira.rax.io/browse/SURF-812

### LGTM's
- [x] Dev LGTM
- [x] Design LGTM
- [x] Zoom LGTM

### CODE REVIEW

File: docs/docs.less [hunk 1]
```diff
@@ -39,34 +42,34 @@
   position: sticky;
   top: 1rem;
 
-  &-title {
-    font-size: 1.5em;
+  > a {
```
Comment (CITguy) [inline docs/docs.less]: fyi: `.article-toc > a`

File: docs/elements/hx-reveal/index.html [hunk 2]
```diff
@@ -3,10 +3,24 @@
 ---
 {% extends 'element.njk' %}
 {% block content %}
+  <section>
+    <p>
+      The custom <code>&lt;hx-reveal&gt;</code> element wraps disclosable content
```
Comment (cathysiller) [inline docs/elements/hx-reveal/index.html]: disclosable? i dont think this is a word, can we reword? disclosed content?

File: docs/elements/hx-tabset/index.html [hunk 3]
```diff
@@ -1,18 +1,42 @@
...
+{% block attributes %}
   <dl>
-    <dt>
-      current-tab <small>{Number=0}</small>
-    </dt>
-    <dd>
-      Zero-based index of the currently active tab + tabpanel.
-    </dd>
+    <dt>current-tab <small>{Number=0}</small></dt>
+    <dd>Zero-based index of the currently active tab + tabpanel.</dd>
```
Comment (cathysiller) [inline docs/elements/hx-tabset/index.html]: yeah some `<dd>` are on one line and others on seperate (even though they can fit on one)- we should fix in another story to cleanup

**Comment by cathysiller** [comment]
dev lgtm

**Comment by araiford** [comment]
design LGTM.

**Pull request merged**